ecoscope/version.py

```diff
@@ -1 +1 @@
-__version__ = "1.0.1"
+__version__ = "1.1.0"
```

Comment:
I think let's bump to 1.0.2 instead as I see this as a bug fix rather than introduction of new functionality

Reply:
This bump was for the previous additions to Ecoscope... I had made the PR to bump to 1.1.0 before I did this change of patrol_id. So for patrol_id, it ought to be bumped to 1.1.2...

Reply:
I have bumped the version to 1.1.1

Reply:
Thanks